fe/fe-core/src/main/java/org/apache/doris/analysis/LateralViewRef.java

```diff
@@ -17,10 +17,10 @@
 
 package org.apache.doris.analysis;
 
+import com.google.common.base.Preconditions;
```

Comment:
pay attention to the order of import package

Reply:
done